google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingImpl.java

```diff
@@ -919,6 +924,54 @@ static ListLogEntriesRequest listLogEntriesRequest(
     return listLogEntriesAsync(getOptions(), optionMap(options));
   }
 
+  static TailLogEntriesRequest tailLogEntriesRequest(
```

Comment:
Is this not private for testing purposes?
I suggest you rename it to createTailLogEntriesRequest or buildTailLogEntriesRequest to follow conventions for builder method names.

Reply:
> Is this not private for testing purposes?
Yes. Similarly to `listLogEntriesRequest()`.
I will rename it. The old habbit to keep the code homogeneous including naming convention dies hard.